### PR TITLE
numbers tasks

### PR DESCRIPTION
**Pull request by Bv28dog**

### CODE REVIEW

File: number_18/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const pythagorean = (a, b) => {
+  const c = Math.sqrt(a * a + b * b);
+  return c;
+};
+console.log(pythagorean(5, 12));
+
+const generatePassword = (n) => {
+  let characters =
```
Comment (cos1715) [inline number_18/index.js:8]: const

File: number_18/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  for (let i = 0; i < n; i++) {
+    let randomIndex = Math.floor(Math.random() * characters.length);
+    password += characters.charAt(randomIndex);
+  }
+  return password;
+};
+console.log(generatePassword(8));
+
+const isPrime = (n) => {
+  let kv = Math.sqrt(n)
```
Comment (cos1715) [inline number_18/index.js:20]: Why do you need kv?

File: number_18/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      decimal: Math.round(decimalPart)
+    };
+    
+    return result;
+  }
+  console.log(splitNumber(2)); 
+  console.log(splitNumber(2.34)); 
+  
+  const isArmstrong = (number) =>  {
+    const [firstStrNum, secondStrNum, thirdStrNum] = number.toString();
```
Comment (cos1715) [inline number_18/index.js:48]: What if number is 1 or 11 or 1111?

**Review by cos1715 — CHANGES_REQUESTED** [review]

File: number_18/index.js (hunk 2 above)
Comment (Bv28dog) [reply number_18/index.js:20]: I was in a hurry, so I called the root square

File: number_18/index.js (hunk 1 above)
Comment (Bv28dog) [reply number_18/index.js:8]: I'm Sorry

File: number_18/index.js (hunk 3 above)
Comment (Bv28dog) [reply number_18/index.js:48]: I'll fix it(

**Pull request merged**